packages/wonder-blocks-typography/util/styles.js

```diff
@@ -15,7 +15,7 @@ const common = {
 const styles = StyleSheet.create({
     Title: {
         ...common,
-        fontFamily: "Lato",
+        fontFamily: ["Lato", "sans-serif"],
```

Comment:
It feels like there's is a lot of repetition in here, but I think this is fine for now.